cookbooks/hdfsdu/templates/default/hdfsdu_service_pom.xml.erb

```diff
@@ -0,0 +1,116 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.twitter.hdfsdu</groupId>
+    <artifactId>hdfsdu</artifactId>
+    <version><%= node[:hdfsdu][:version] %></version>
+    <relativePath>..</relativePath>
+  </parent>
+  <groupId>com.twitter.hdfsdu</groupId>
+  <artifactId>hdfsdu-service</artifactId>
+  <packaging>jar</packaging>
+  <version><%= node[:hdfsdu][:version] %></version>
+  <name>service</name>
+  <url>http://maven.apache.org</url>
+
+  <licenses>
+    <license>
+      <name>Apache License</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0</url>
+    </license>
+  </licenses>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
```

Comment:
Should the maven-compiler-plugin change to the POM be upstreamed? If so, then a simple XSLT could transform the versions to what we want without forking the HDFSDU POM.